Fix compilation on ios and provide NopClipboardContext on it (#2)
alacritty/copypasta: CHANGELOG.md

# Changelog
All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

> 

Reviewer: ```suggestion
```

## Unreleased

### Fixed

> - Fix compilation on ios and provide NopClipboardContext on it

Reviewer: ```suggestion
- Compilation on iOS, using the no-op clipboard
```
